freezeyt.py

```diff
@@ -1,18 +1,30 @@
+from urllib.parse import urlparse
+
 import xml.dom.minidom
 
 import html5lib
 
-
 def url_to_filename(base, url):
-    """Return the filename to which the page is frozen.
-
-    base -- path to the file
-    url -- web app endpoint of the page
     """
+    Returns full file system path from parsed absolute or relative URL
+    base - Filesystem base path (eg. /tmp/)
+    url - Absolute or relative URL (eg. http://localhost:8000/ or /second/second.html)
+    """
+    url_parse = urlparse(url)
+
+    if url.startswith('http'):
```

Comment:
```suggestion
    if url.startswith('http') or url.startswith('https'):
```

Reply:
Maybe we could add 'https' protocol also.

Reply:
If the URL string `startswith('http')`, it also `startswith('https')`.
But, you can use `url_parse.scheme` instead of `startswith`.